#Standard header
Header header

#######################
# Debug 60000-60019
#######################
uint16 register_60000
uint16 register_60001
uint16 register_60002
uint16 register_60003
uint16 register_60004
uint16 register_60005
uint16 register_60006
uint16 register_60007
uint16 register_60008
uint16 register_60009
uint16 register_60010
uint16 register_60011
uint16 register_60012
uint16 register_60013
uint16 register_60014
uint16 register_60015
uint16 register_60016
uint16 register_60017
uint16 register_60018
uint16 register_60019
